src/main/java/com/renatusnetwork/momentum/commands/PlotCMD.java

```diff
@@ -294,57 +296,66 @@ private void deletePlot(Plot targetPlot, boolean forceCleared, Player player) {
 
     private void untrustPlayer(Player player, String[] a)
     {
-        PlayerStats target = Momentum.getStatsManager().getByName(a[1]);
+        Plot plot = Momentum.getPlotsManager().get(player.getName());
 
-        if (target != null)
-        {
-            Plot plot = Momentum.getPlotsManager().get(player.getName());
+        // if you have plot
+        if (plot == null) {
+            player.sendMessage(Utils.translate("&cYou do not have a plot"));
+            return;
+        }
 
-            // if you have plot
-            if (plot != null)
-            {
-                // if they are not a trusted player
-                if (plot.isTrusted(target.getUUID()))
-                {
-                    Momentum.getPlotsManager().removeTrusted(plot, target);
-                    player.sendMessage(Utils.translate("&7You removed &3" + target.getDisplayName() + " &7from your plot!"));
+        String name = a[1];
+        new BukkitRunnable() {
+            @Override
+            public void run() {
+                PlayerStats playerStats = Momentum.getStatsManager().getByName(name);
+                String target = playerStats == null ? StatsDB.getUUIDByName(name) : playerStats.getUUID();
+                if (target == null) {
+                    player.sendMessage(Utils.translate("&4" + name + " &chas never played before"));
+                    return;
                 }
-                else
-                    player.sendMessage(Utils.translate("&4" + target.getDisplayName() + " &cis not trusted in your plot"));
-            }
-            else
-                player.sendMessage(Utils.translate("&cYou do not have a plot"));
-        }
-        else
-            player.sendMessage(Utils.translate("&4" + a[1] + " &cis not online!"));
 
+                if (!plot.isTrusted(target)) {
+                    player.sendMessage(Utils.translate("&4" + name + " &cis not trusted in your plot"));
+                    return;
+                }
+
+                Momentum.getPlotsManager().removeTrusted(plot, target);
+                player.sendMessage(Utils.translate("&7You removed &3" + name + " &7from your plot"));
```

Comment:
Nit picky but "You removed" to "You untrusted", better wording

Reply:
changed